cost/aws/savings_plan/expiration/aws_savings_plan_expiration.pt

```diff
@@ -0,0 +1,175 @@
+name "AWS Expiring Savings Plans"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications before AWS Savings Plan expire. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/reserved_instances/expiration) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
```

Comment:
think the first link is incorrect - should be
`See the [README](https://github.com/flexera/policy_templates/tree/master/cost/aws/savings_plan/expiration)`